release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -3415,11 +3415,13 @@ $ oc adm release info 4.15.18 --pullspecs
 The following enhancements are included in this z-stream release:
 
 [id="ocp-4-15-18-unservable-future-versions-status-condition"]
-===== Introducing the UnservableFutureVersions status condition for SHA1 routes
+===== Blocking upgrades for SHA1 serving certificates on routes and Ingress controllers
 
-* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} causes routes with SHA1 certificates to be rejected.
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to 4.16 cause routes and Ingress controllers with SHA1 certificates to be rejected.
```

Comment:
This release note relates to IngressControllers objects, not controllers for Ingress objects, so it should be written "Ingress Controllers" per the glossary.  (Naming is hard.)  This comment applies to all 4 occurrences (once in the heading and thrice in the text under the heading).  
https://github.com/openshift/openshift-docs/blob/230e6dd459166b0b3b7af998bd059025e166f238/contributing_to_docs/term_glossary.adoc?plain=1#L377-L382